src/frontend/scss/components/templates/courses/cms/_blogpost_detail.scss

```diff
@@ -131,7 +131,7 @@
         left: -0.6rem;
```

Comment:
So, change the way the Glimpse renders? I'm all for simplicity and I think it makes sense.

Reply:
I kinda like what Medium does with categories and tags
![image](https://user-images.githubusercontent.com/34194007/167446147-c37cd727-b123-4e4a-9f64-04936e13818a.png). We could do this in article detail.

We could do a "Check out other {{ category }} of find new {{categories separated by commas}}". It's a more conversational approach that (IMHO) makes more sense to go for.

Reply:
I guess now it's clear when using a screen reader because it's announced like:

```
level 1 heading News, Categories, link Beginner, link Advanced, link Expert, ..., level 1 heading Robust systemic framework, Categories, link Expert, link Event, ...
 ```

But visually yeah it could use some clarification. Just having a "Filter by categories" text before the category tags would be enough I guess.

Reply:
I guess if you want to take time to decide what to do, this can be handled in a future PR?

Reply:
A quick overview to visualize both solutions:
### Prepend "Filter by categories" before tags :
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/9265241/167594277-ca1ed005-11aa-49a7-9487-2352a9e572f4.png">

### Move categories to the right side:
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/9265241/167595665-b16534b1-a60e-4080-9e32-2c9173c44873.png">

Reply:
I think the categories on top of the detail blog as it is now does not make sense because it feels that you propose to filter the current page when in fact you are proposing discovery navigation....
+1 on the side bar version with a title even more explicit like "Other categories of articles"; WDYT?

Reply:
+1 as well on categories on the sidebar. It's additional content that makes sense to be on a secondary space.

Reply:
Ok, so let's move those categories to the right side. Let's address this in another PR.